app/views/case_workers/admin/management_information/index.html.haml

```diff
@@ -20,68 +20,25 @@
       = govuk_link_button(t('.regenerate'), case_workers_admin_management_information_generate_url(report_type: report_type))
 
 #provisional-assessment-date.form-section.fx-dates-chooser
-  .form-group
-    %h2.heading-large
-      = t('.provisional_assessments_by_date')
-    .form-group
-      .report-status
-        = hidden_field_tag :user_api_key, @current_user.api_key
-        = t('.report_information')
+  %h2.heading-large
+    = t('.provisional_assessments_by_date')
 
-    -# GDS Date start
-    .form-group.fx-start-date
-      %fieldset
-        %legend
-          %span.form-label-bold
-            = t('.start_date')
-          %span.form-hint
-            = t('.example_date')
+  = form_with url: '', builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f|
```

Comment:
is the the `url: ''` needed? - default [form_with url is nil](https://api.rubyonrails.org/v6.1/classes/ActionView/Helpers/FormHelper.html#method-i-form_with) which [will POST to the current page](https://guides.rubyonrails.org/form_helpers.html#dealing-with-basic-forms) according to that guide - but have not tested